dev/gallery.tsx

```diff
@@ -107,7 +107,10 @@ export function Gallery() {
                 </nav>
             </header>
             <main className={css(styles.main)}>
-                <View style={styles.cards}>
+                <View
+                    style={styles.cards}
+                    className={isMobile ? "perseus-mobile" : ""}
```

Comment:
This is the magic to flip Perseus into mobile rendering for some widgets (used in our `.less` files mostly).